examples/3_NeuralNetworks/autoencoder.py

```diff
@@ -83,11 +86,51 @@ def decoder(x):
 optimizer = tf.train.RMSPropOptimizer(learning_rate).minimize(cost)
 
 # Initializing the variables
-init = tf.initialize_all_variables()
+init = tf.global_variables_initializer()
```

Comment:
Did you rebase your local branch before merging and creating pull request?
These initializer functions are different in the master branch as they have been updated for TensorFlow 1.0